.travis.yml

```diff
@@ -5,6 +5,10 @@ bundler_args: --without development
 rvm:
   - "2.1.5"
 
+branches:
+  only:
+  - master
```

Comment:
One more idea: We give CI it's own repository to build stuff. That would also mean more resources and our main repository is free from `merged-pull/*` branches.